App/src/services/dossier.ts

```diff
@@ -42,3 +43,7 @@ export function getDossierRequiredReview(): Promise<GetMyDossiersResponse> {
 export function getDossierReport(id: string): Promise<DossierReportResponse> {
     return axios.get(`/Dossier/GetDossierReportByDossierId/${id}`);
 }
+
+export function searchDossiers(title: string, state: DossierStateEnum, guid: string): Promise<GetMyDossiersResponse> {
+    return axios.get("/Dossier/SearchDossiers?title=" + title + "?state=" + state + "?groupId=" + guid);
```

Comment:
I haven't tested it, but does this even work as expected? Query parameters are normally separated by &, not ?, is it not ill-formatted?

As we had discussed together, all these parameters are optional and will not necessarily be specified by the user (for example, they might want all approved dossiers without regards to anything else). Indeed, in the case where state is anything but InReview, a groupId should not even be passed in. That's why I had shared this link with you in our private chat, to construct a query based on the supplied arguments https://stackoverflow.com/a/49701878